SCClassLibrary/Common/Core/Kernel.sc

```diff
@@ -596,6 +628,16 @@ Method : FunctionDef {
 		^[names, values].flop.flatten
 	}
 
+	isClassMethod {	^ownerClass.isMetaClass	}
+
+	evaluateWithArgsAndKwArgs{|argsArray, keywordArgsEvent, thisObject|
```

Comment:
why `evaluate` and not `perform`?

Reply:
Since this doesn't take a selector, `valueWithArgsAndKeywordArgs` would be the option,
however, this would require that I change all `perform*` methods to actually evaluate the method rather than looking up other methods inside of `Method`, while `Method` isn't the most common class, I was trying to break as little code as possible. 

I don't know that it makes sense to have `Method` be a true 'callable' object as it requires error checking the type of 'this' against the class — otherwise there would be very hard to find bugs.

Reply:
You can query the argNames and defaultArgs (prototypeFrame) of any method like you can for a function definition. I don't think you need all this complication.